bitnami/jaeger/templates/NOTES.txt

```diff
@@ -0,0 +1,23 @@
+CHART NAME: {{ .Chart.Name }}
+CHART VERSION: {{ .Chart.Version }}
+APP VERSION: {{ .Chart.AppVersion }}
+
+** Please be patient while the chart is being deployed **
+
+1. Get the application URL by running these commands:
+
+{{- if contains "NodePort" .Values.query.service.type }}
+    export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "common.names.fullname" . }})
+    export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+    echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.query.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "common.names.fullname" . }}'
+    export SERVICE_IP=$(kubectl get svc --namespace {{ include "jaeger.query.fullname" }} {{ include "common.names.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    echo http://$SERVICE_IP:{{ .Values.query.service.ports.api }}
+{{- else if contains "ClusterIP" .Values.query.service.type }}
+    echo "Browse to http://127.0.0.1:8080"
+    kubectl port-forward svc/{{ include "common.names.fullname" . }} 8080:{{ .Values.query.service.ports.api }} &
+{{- end }}
+
+{{- include "common.warnings.rollingTag" .Values.image }}
```

Comment:
We normally add a call to the validateValues helper, where you would add some very basic checks. For example, I would check if any of cassandra or the external database are enabled (check Appsmith for an example)

Reply:
[fdfd82b](https://github.com/bitnami/charts/pull/13480/commits/fdfd82bbc86311c29a1246fcb7414672e1cf6536)